tools/trim_galore/trim_galore.xml

```diff
@@ -48,8 +48,8 @@
         <requirement type="package" version="1.8.3">cutadapt</requirement>
         <requirement type="package" version="1.8">cutadapt</requirement>
     </requirements>
-    <version_command>
-        perl $__tool_directory/trim_galore --version
```

Comment:
Ah, the original `<version_command>` gives the following error message: `Can't open perl script "/trim_galore": No such file or directory`. My guess is that `$__tool_directory` isn't set.